dir.targets

```diff
@@ -43,6 +43,7 @@
     <!-- if building desktop facade, we don't bin place the refs -->
     <BinPlaceRef Condition="'$(BinPlaceRef)' == '' And '$(BuildingDesktopFacade)' != 'true' And ('$(IsReferenceAssembly)' == 'true' OR '$(IsRuntimeAndReferenceAssembly)' == 'true')">true</BinPlaceRef>
     <BinPlaceRuntime Condition="'$(BinPlaceRuntime)' == '' And ('$(IsRuntimeAssembly)' == 'true' OR '$(IsRuntimeAndReferenceAssembly)' == 'true')">true</BinPlaceRuntime>
+    <BinPlaceRuntime Condition="'$(BinPlaceRuntime)' == '' And ('$(IsNETCoreApp)' == 'true' AND '$(IsTestProject)' == 'true' AND '$(BinPlaceNETCoreAppTests)' == 'true')">true</BinPlaceRuntime>
```

Comment:
If that is what you need we can probably add some option that will filter and only build stuff that is NETCoreApp but that would be at a different layer.  I can help you later doing some filtering, probably somewhere in https://github.com/dotnet/corefx/blob/master/buildvertical.targets.